src/vs/workbench/contrib/chat/browser/contrib/chatInputEditorContrib.ts

```diff
@@ -47,10 +51,10 @@ class InputEditorDecorations extends Disposable {
 	}
 
 	private updateRegisteredDecorationTypes() {
-		const theme = this.themeService.getColorTheme();
 		this.codeEditorService.removeDecorationType(slashCommandTextDecorationType);
+		this.updateInputEditorContentWidgets({ hide: true });
 		this.codeEditorService.registerDecorationType(decorationDescription, slashCommandTextDecorationType, {
-			color: theme.getColor(textLinkForeground)?.toString()
+			opacity: '0'
```

Comment:
This feels kind of suspicious to me but I don't know how else to hide the leading `/` in the slash command text (which needs to actually be in the input model, any pointers @roblourens?